tools/open-test-database-shell.sh

```diff
@@ -9,9 +9,12 @@ case $db in
     pgsql)
         docker exec -e PGPASSWORD=password -it mongooseim-pgsql psql -U postgres -d ejabberd -h 127.0.0.1
     ;;
-    mssql)
+    mssql-sqlcmd)
```

Comment:
Why do we need both scripts? What was wrong with the previous one?

Reply:
sqlcmd is very hard to use, but built in.
sql-cli is better, still does not support multi-line queries, but at least it's more usable.